pom.xml

```diff
@@ -26,7 +26,7 @@
     <maven.compiler.release>11</maven.compiler.release>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <quarkus.version>3.2.2.Final</quarkus.version>
+    <quarkus.version>999-SNAPSHOT</quarkus.version>
```

Comment:
Hmmm, no you shouldn't do that.

What you need is to set up Ecosystem CI.

Reply:
Do you mean this https://github.com/quarkiverse/quarkiverse/issues/115? Actually, I did that to be able to fix the problem raised by the CI https://github.com/quarkiverse/quarkiverse/issues/115#issuecomment-1647236680. Is there a better way to fix a problem that only occurs with the latest snapshot?

Reply:
In other words, my problem here is not to be able to test against the latest snapshot but more to fix my code to make the build pass against the latest snapshot

Reply:
Ah OK, I understand why you did that better. No there's no better way in this case.

Reply:
The problem here is that now you can't simply release it when you feel like it, as you can't have SNAPSHOT dependencies in your released artifact. I'd recommend having `main` compatible with 3.2.0 and hold any breaking changes until the Quarkus release containing those changes is released

Reply:
Actually, adding the dependency to `org.graalvm.sdk:graal-sdk` might have already fixed your problem, so I think you can revert the Quarkus core dependency here to 3.2.0.Final again

Reply:
> The problem here is that now you can't simply release it when you feel like it, as you can't have SNAPSHOT dependencies in your released artifact. 

Yes, I'm aware of that, I already did the exact same thing before, see https://github.com/quarkiverse/quarkus-groovy/pull/36 but it remains a small change to revert before the next release

> I'd recommend having `main` compatible with 3.2.0 and hold any breaking changes until the Quarkus release containing those changes is released

Yes but If you do so, you get a new message indicating that your build fails every day until the next Quarkus minor release which is quite annoying moreover I don't want to discover all migration problems at the very last minute when migrating, I prefer to fix the problems asap when they appear.

IMHO, it should be the exact opposite instead of enforcing tag versions in the main branch of Quarkiverse extensions, we should only have snapshots and the GH action used for the release should allow us somehow to set the Quarkus tag, this way, we would be able to fix easily the migration problems when they come.

Reply:
> Actually, adding the dependency to `org.graalvm.sdk:graal-sdk` might have already fixed your problem, so I think you can revert the Quarkus core dependency here to 3.2.0.Final again

Yeah, I know it is the fix actually

Reply:
Sorry about the noise btw :)